packages/@webex/plugin-meetings/src/meeting/util.ts

```diff
@@ -55,7 +55,10 @@ const MeetingUtil = {
       );
     }
 
-    Metrics.postEvent({event: eventType.MEDIA_REQUEST, meeting});
+    NewMetrics.submitClientEvent({
```

Comment:
we should probably move the code for sending 'client.locus.media.request' and 'client.locus.media.response' into LocusMediaRequest.sendHttpRequest() function because that's when the /media request is actually being sent out (we can do it in another PR)

Reply:
Have created a jira for this: https://jira-eng-gpk2.cisco.com/jira/browse/SPARK-440552